docs/fiddles/features/progress-bar/main.js

```diff
@@ -3,14 +3,26 @@ const { app, BrowserWindow } = require('electron')
 function createWindow () {
   const win = new BrowserWindow({
     width: 800,
-    height: 600,
-    webPreferences: {
-      nodeIntegration: true
-    }
+    height: 600
   })
 
   win.loadFile('index.html')
-  win.setProgressBar(0.5)
+  
+  const INCREMENT = 0.02;
+  const INTERVAL_DELAY = 100; //ms
+  let c = 0;
+  let interval = setInterval(() => {
+    
+    //update progress bar to next value
+    win.setProgressBar(c);
+
+    if(c > 1) {
+      //progress bar has reached full so reset it and stop the timer
+      win.setProgressBar(-1);
+      clearTimeout(interval);
+    }
+    c += INCREMENT;
+  }, INTERVAL_DELAY);
 }
```

Comment:
We don't have linting set up for `/docs/fiddles`, but could you lint this file with the https://standardjs.com/ linter default?

Reply:
Implemented in commit `ca9a`